src/components/assets/icons/SSIEntityIcon/index.tsx

```diff
@@ -1,29 +1,41 @@
 import React, { FC } from 'react'
```

Comment:
lets rename this component to SSIProfileIcon, to maker it more clear what this component is for

Reply:
renamed